src/templates/example.graphml

```diff
@@ -52,6 +53,7 @@
     <node id="11">
       <data key="version">vasild/bitcoin#relay_tx_to_priv_nets</data>
       <data key="bitcoin_config">sensitiverelayowntx=1,uacomment=sensitive_relay</data>
+      <data key="build_args">--disable-zmq --disable-fuzz</data>
```

Comment:
if you put this in `test/data/build_v24_test.graphml` we could actually see the options in the test log:

https://github.com/bitcoin-dev-project/warnet/actions/runs/6544930178/job/17772400409?pr=145#step:6:318